compiler/codegen/cg-expr.cpp

```diff
@@ -2945,6 +2946,10 @@ GenRet codegenZero32()
   return new_IntSymbol(0, INT_SIZE_32)->codegen();
 }
 
+static GenRet codegenString(const char* val) {
```

Comment:
Perhaps we should call it `codegenCString` ?

Reply:
Agreed. Especially given the confusion I had over the Chapel vs C strings earlier. I've updated it; thanks!